tendrl/gluster_integration/message/callback.py

```diff
@@ -489,8 +489,7 @@ def volume_delete(self, event):
             if fetched_volume.name == event['message']['name']:
                 fetched_volume.deleted = True
                 fetched_volume.deleted_at = time_utils.now()
-                sync_ttl = int(NS.config.data.get("sync_interval", 10))
-                fetched_volume.save(ttl=sync_ttl)
+                fetched_volume.save()
```

Comment:
Let the volume deleted by TTL which assigns from sync thread, because if a cluster is big and volume is very high then assigning 60 sec of default sync interval to delete is not good, because it may delete volume in between of sync.